pages/s/[org]/[repo]/contributors.tsx

```diff
@@ -221,7 +222,10 @@ export default function RepoPageContributorsTab({ repoData, ogImageUrl }: RepoPa
                 <h2 className="text-lg font-semibold">Contributors</h2>
                 <LayoutToggle
                   value={contributorsView}
-                  onChange={() => setContributorsView((prev) => (prev === "list" ? "grid" : "list"))}
+                  onChange={() => {
+                    setContributorsView((prev) => (prev === "list" ? "grid" : "list"));
```

Comment:
refactor (blocking): no need to use `useState` since we're using the querystring now. You can use `setQueryParams` to updated the querystring.

https://github.com/open-sauced/app/blob/ccdd4fbd5131c2910ba59024942728e7c32c56a2/pages/workspaces/%5BworkspaceId%5D/contributor-insights/%5BlistId%5D/overview.tsx#L111-L115